CRM/Core/xml/Menu/Admin.xml

```diff
@@ -262,15 +262,15 @@
      <page_callback>CRM_Admin_Page_MessageTemplates</page_callback>
      <adminGroup>Communications</adminGroup>
      <icon>admin/small/template.png</icon>
-     <access_arguments>edit message templates</access_arguments>
+     <access_arguments>edit message templates;edit user-driven message templates;edit system workflow message templates</access_arguments>
```

Comment:
I see the xml sometimes uses commas & sometimes semi-colons - do they have the same meaning?

Reply:
I think it’s the diff between or and I think ; == or , == and

Reply:
I think Seamus is [right](https://docs.civicrm.org/dev/en/latest/framework/routing/#access_arguments)